docs/source-pytorch/data/infinite.rst

```diff
@@ -0,0 +1,49 @@
+:orphan:
+
+Iterable Datasets
+=================
+
+Lightning supports using :class:`~torch.utils.data.IterableDataset` as well as map-style Datasets. IterableDatasets provide a more natural
+option when using sequential data.
```

Comment:
I wouldn't say sequential data, because that refers to training on sequence data and for this it doesn't really matter what data iter type we use.

I would say probably what was meant here is "streaming data". Maybe this was copied from the previous docs.